src/streamlink/stream/rtmpdump.py

```diff
@@ -21,7 +21,7 @@ class RTMPStream(StreamProcess):
     - :attr:`params` A :class:`dict` containing parameters passed to rtmpdump
     """
 
-    __shortname__ = "rtmp"
+    __shortname__ = "rtmpdump"
```

Comment:
Changing the Stream class's shortname is a breaking change, as it changes the expected/documented values of the player-passthrough and related parameters:
https://streamlink.github.io/cli.html#cmdoption-player-passthrough

Reply:
True that